### DISCUSSION TITLE
Plotting the time varying model coefficients for regression models (eg. in MMM)

### DISCUSSION THREAD

**Discussion by rjscout24** [uber/orbit#800] [Q&A]
In the paper it is mentioned that the model coefficients are time varying. How do we plot this or get this data? I did not see any example code for this. I did see codes for regression but that just provided single values - are these calculated based on mean across all time steps?

**Comment by oliver-kutis** [comment]
> In the paper it is mentioned that the model coefficients are time varying. How do we plot this or get this data? I did not see any example code for this. I did see codes for regression but that just provided single values - are these calculated based on mean across all time steps?
> 
> 

There is a method available after you've fitted the model called "get_regression_coefs"
